src/queries/get-all-strategies-data.graphql

```diff
@@ -1,7 +1,20 @@
-query GetAllStrategiesData {
+query GetAllStrategiesData($monthoAgoEpoch: Int!) {
```

Comment:
is anything else using this query? should we make this change a new file?

Reply:
not as far as i can tell